### ISSUE TITLE
Invalid arguments to vboxmanage (potentially malformed path)

### ISSUE THREAD

**Issue by raulsimpat** [hashicorp/vagrant#10832]
### Vagrant version
2.2.4

### Host operating system
Ubuntu 14.04

### Guest operating system
Windows 10

### Vagrantfile
```
# -*- mode: ruby -*-
# vi: set ft=ruby :

Vagrant.configure("2") do |config|
  config.vm.box = "hashicorp/precise64"
end
```

### Debug output
==> default: Importing base box 'hashicorp/precise64'...
There was an error while executing `VBoxManage`, a CLI used by Vagrant
for controlling VirtualBox. The command and stderr is shown below.

Command: ["import", "\\\\?\\C:\\Users\\jimenezjrs\\.vagrant.d\\boxes\\hashicorp-VAGRANTSLASH-precise64\\1.1.0\\virtualbox\\box.ovf", "--vsys", "0", "--vmname", "precise64_1557187833529_48843", "--vsys", "0", "--unit", "12", "--disk", "C:/Users/JuanRa\u00B7lJim\u0398nezSoto/VirtualBox VMs/precise64_1557187833529_48843/box-disk1.vmdk"]

Stderr: 0%...10%...20%...30%...40%...50%...60%...70%...80%...90%...100%
Interpreting \\?\C:\Users\jimenezjrs\.vagrant.d\boxes\hashicorp-VAGRANTSLASH-precise64\1.1.0\virtualbox\box.ovf...
OK.
0%...
Progress state: E_INVALIDARG
VBoxManage.exe: error: Appliance import failed
VBoxManage.exe: error: Code E_INVALIDARG (0x80070057) - One or more arguments are invalid (extended info not available)
VBoxManage.exe: error: Context: "enum RTEXITCODE __cdecl handleImportAppliance(struct HandlerArg *)" at line 957 of file VBoxManageAppliance.cpp

### Expected behavior
start and provision the virtual machine

### Actual behavior
I tried to start the virtual machine by doing vagrant up but there was a problem with my username (it had some accents). I changed the folder name and a registry value for it to work. But after doing that the problem as stated by the output appears. It looks like it's using the previous username where it should update this value automatically?

### Steps to reproduce
1. Change a windows user folder to another value.
2. Try to do vagrant up.

**Comment by chrisroberts** [comment]
Hi there,

This seems to show the underlying issue:

> Interpreting \?\C:\Users\jimenezjrs.vagrant.d\boxes\hashicorp-VAGRANTSLASH-precise64\1.1.0\virtualbox\box.ovf...

where `jimenezjrs.vagrant.d` != `jimenezjrs\.vagrant.d`. You might try either deleting the `.vagrant.d` directory in your user's home directory, or moving it if you want to retain the content within it. Then running `vagrant up` again and see if the incorrect path is still used.

**Comment by briancain** [comment]
Hey there,

I am going to close this due to lack of response. If this is still occurring, please open a new issue and follow the provided issue template that appears when you click the "New Issue" button. This will help us in getting a reproduction and fix. Thanks!

**Comment by vardhireddy** [comment]
Facing the same issue..
D:\SDNProject\bp_eval_1902>vagrant up
Bringing machine 'default' up with 'virtualbox' provider...
==> default: Importing base box 'bp-eval-1902'...
Progress: 90%There was an error while executing `VBoxManage`, a CLI used by Vagrant
for controlling VirtualBox. The command and stderr is shown below.

Command: ["import", "\\\\?\\C:\\Users\\krishnareddy.v\\.vagrant.d\\boxes\\bp-eval-1902\\0\\virtualbox\\box.ovf", "--vsys", "0", "--vmname", "bpee_BPEval19022_1556762107721_88022_1567163831401_46229", "--vsys", "0", "--unit", "12", "--disk", "C:/Users/krishnareddy.v/VirtualBox VMs/bpee_BPEval19022_1556762107721_88022_1567163831401_46229/box-disk001.vmdk"]

Stderr: 0%...10%...20%...30%...40%...50%...60%...70%...80%...90%...100%
Interpreting \\?\C:\Users\krishnareddy.v\.vagrant.d\boxes\bp-eval-1902\0\virtualbox\box.ovf...
OK.
0%...10%...20%...30%...40%...50%...60%...70%...80%...90%...
Progress state: E_INVALIDARG
VBoxManage.exe: error: Appliance import failed
VBoxManage.exe: error: Code E_INVALIDARG (0x80070057) - One or more arguments are invalid (extended info not available)
VBoxManage.exe: error: Context: "enum RTEXITCODE __cdecl handleImportAppliance(struct HandlerArg *)" at line 957 of file VBoxManageAppliance.cpp

**Comment by tonykarre** [comment]
@raulsimpat @chrisroberts @briancain @vardhireddy  I also had this same problem. Deleting the .vagrant.d folder did not work by itself. 

I opened the VirtualBox Manager app and saw that I had a huge collection of vagrant-created VMs that were essentially orphaned, probably because I had not been doing a "vagrant destroy" and had simply been deleting the data from my vagrant folder.  So I did the following:
1.  In VirtualBox Manager, "removed" all of the vagrant VMs
2.  deleted the %userprofile%\.vagrant.d folder
3.  cleaned out my local vagrant folders (i.e., the folders from where you "vagrant init")

This big cleanup fixed the problem.
